.github/workflows/_custom-e2e-test.yaml

```diff
@@ -22,16 +19,19 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        backend-name: ${{ fromJson(inputs.backends) }}
+        backend-name:
+          - Stubbed
+          - QX emulator
     env:
       E2E_USERNAME: ${{ secrets.e2e_username }}
       E2E_PASSWORD: ${{ secrets.e2e_password }}
     steps:
       - uses: actions/checkout@v4
-      - uses: qutech-delft/github-actions/.github/actions/poetry-setup
+      - uses: QuTech-Delft/GitHub-Actions/poetry-setup@master
         with:
           python-version: ${{ inputs.python-version }}
           poetry-version: ${{ inputs.poetry-version }}
+      - uses: actions/checkout@v4
       - name: Install packages
         run: poetry install
       - name: Run e2e test
```

Comment:
Probably should have noticed this in an earlier PR, but (for the line below) shouldn't the path tests/e2e_test.py be an input to the workflow instead of a hardcoded path?

Reply:
I'll make it an input

Reply:
Done